cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -19,16 +19,18 @@ package options
 import (
 	"fmt"
 	"io/ioutil"
-	"k8s.io/klog/v2"
 	"os"
 
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
 	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 	kubeschedulerscheme "k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
 	kubeschedulerconfigv1beta1 "k8s.io/kubernetes/pkg/scheduler/apis/config/v1beta1"
 )
 
-func loadConfigFromFile(file string) (*kubeschedulerconfig.KubeSchedulerConfiguration, error) {
+// LoadConfigFromFile will load the file passed in and decode the file into `config.KubeSchedulerConfiguration`.
+// This method is used in two places (main scheduler bootstrapping, scheduler_perf) in order to build ComponentConfig.
+func LoadConfigFromFile(file string) (*kubeschedulerconfig.KubeSchedulerConfiguration, error) {
```

Comment:
I don't think we want this dependency. There is no much logic, you can duplicate.

Reply:
+1, this dependency is strange.

Reply:
I suggested this code reuse in an earlier review :) we also export LogOrWriteConfig for use in a different package within kube-scheduler. anyway I don't feel strongly about this either way.

Reply:
well, now that we have more voice to drop the dependency,  I will revert this back.

Reply:
Done